lib/application.js

```diff
@@ -241,9 +241,11 @@ class Application extends api.events.EventEmitter {
         return;
       } else if (placeName === 'schemas') {
         const dir = api.path.join(this.dir, 'schemas');
-        api.metaschema.fs.loadAndCreate(dir, null, (error, schemas) => {
+        const provider = api.gs.provider;
+        api.metaschema.fs.loadAndCreate(dir, provider, (error, schemas) => {
```

Comment:
Passing provider as a second argument here, it will be resolved to the `api` field in the `global` object inside the schema execution context.

Reply:
I'll pass `application.sandbox` as a second argument